src/components/Header/Header.jsx

```diff
@@ -1,78 +1,92 @@
-import React, { useState } from 'react';
-import { useNavigate } from 'react-router-dom';
+import React, { useEffect, useState } from 'react';
+import { Link } from 'react-router-dom';
+import HeaderMenu from './componenets/HeaderMenu';
 import styled from 'styled-components';
+import { flexSort } from '../../styles/mixin';
 
 const Navbar = () => {
   const [isLoggedIn, setIsLoggedIn] = useState(false);
-  const [username, setUsername] = useState('');
-  const navigate = useNavigate();
+  const [isMenuOpen, setIsMenuOpen] = useState(false);
+  const TOKEN = localStorage.getItem('TOKEN');
 
-  const handleLogin = () => {
-    //TODO: 로그인 로직 구현
-    setIsLoggedIn(true);
-    setUsername('user');
+  const showMore = () => {
+    setIsMenuOpen(!isMenuOpen);
```

Comment:
```suggestion
    setIsMenuOpen(prev => !prev);
```
함수형 업데이트 방식으로 state를 처리하는 것이 좋습니다

Reply:
함수형 업데이트로 변경했습니다!